specification/protocol/otlp.md

```diff
@@ -382,8 +416,9 @@ numbers or strings are accepted when decoding.
 
 #### OTLP/HTTP Response
 
-Response body MUST be the appropriate serialized Protobuf message (see below for
-the specific message to use in the Success and Failure cases).
+The response body MUST be the appropriate serialized Protobuf message (see below for
```

Comment:
Can we add a link to `below`?

Reply:
I already added links to the individual `Success`, `Partial` and `Failures`. It's a bit further after `see below`